pkg/unvanquished_src.dpkdir/presets/graphics/high.cfg

```diff
@@ -15,7 +15,7 @@ seta r_imageMaxDimension 2048
 seta r_ext_texture_filter_anisotropic 8
 seta r_textureMode "GL_LINEAR_MIPMAP_LINEAR"
 seta r_clear 1
-seta r_fastsky 0
+seta r_fastsky off
```

Comment:
Yeah but why not? My idea is that we progressively migrate cvars to new style ones and presets when we can, and then once there is no legacy cvar anymore, we will modify he RmlUi checkbox to switch between on/off instead of 0/1. This change is pure bikeshedding, is not required, but why not. When I add new-style boolean cvars to presets I already use on/off, so I prefer to unify them.

Reply:
Well 0/1 is just more reliable, it always works. Old-style cvars, new booleans, or things that used to be booleans and got changed to something with 3 options, 0/1 works with any. Making a word format the default serialization is a mistake if you ask me.